src/redux/slices/reset.ts

```diff
@@ -0,0 +1,13 @@
+import { createAction } from '@reduxjs/toolkit';
+
+// a global action creator
+// other reducers can use this action to reset the state. via `extraReducer`
+// example usage can be check in `src/redux/slices/theme.ts`
+
+// reference for `extraReducer`
+// - https://redux-toolkit.js.org/usage/usage-guide#use-with-redux-persist search for `extraReducer`
+// - https://redux-toolkit.js.org/api/createslice#extrareducers
+
+// current usage
+// - currently being used in `SettingsDrawer/ResetButton.tsx`
+export default createAction('resetState');
```

Comment:
yeah, this shouldn't be reset, this should be settingsReset. Because we're not resetting the entire state, we're resetting particular fields for the settings. The bookmarks slice for e.g. wouldn't reset